packages/http/src/validator/utils/spec.ts

```diff
@@ -1,4 +1,4 @@
-import { IHttpResponse as IHttpResponseSpec } from '@stoplight/types/http';
+import { IHttpOperationResponse as IHttpResponseSpec } from '@stoplight/types/http-spec';
```

Comment:
Need to update errors after changing the import. I am concerned that changing the import is confusing, using findResponseSpec that returns an array of IHttpResponseSpec[] looks like a new type when it isn't.

Also, suggest changing the name of findResponseSpec to findResponseOperation?
Also, suggest changing responseSpecs: IHttpOperationResponse[] to responseOperations: ...
```ts
import { IHttpOperationResponse } from '@stoplight/types';
```

Reply:
Done